### ISSUE TITLE
Path param PHPDoc not working as expected

### ISSUE THREAD

**Issue by slnw** [dedoc/scramble#150]
Hi, when adding
```
    /**
     * @param string $plan
     */
      public function...


```
The document is still generated as Integer instead of string. Attached screenshot.
![Screenshot 2023-07-05 at 19 03 20](https://github.com/dedoc/scramble/assets/26124406/dbde9e2d-b03c-4e68-a1ad-36f99f104450)

route file
![Screenshot 2023-07-05 at 19 08 12](https://github.com/dedoc/scramble/assets/26124406/df25bac7-bfb8-4e94-8d9b-427bd7ab4507)

**Comment by romalytvynenko** [comment]
@slnw thanks for reporting! Will fix.

Quick question, is it `uuid` under the hood?

**Comment by slnw** [comment]
Yes, it's a UUID

**Comment by slnw** [comment]
@romalytvynenko - can you estimate when it'll be fixed?

**Comment by romalytvynenko** [comment]
@slnw I plan to release the next version by the end of July. In the next version I've added UUID support

**Comment by romalytvynenko** [comment]
Closing for now, as UUID support has been implemented in https://github.com/dedoc/scramble/pull/122 and will be released within 0.8.0 release soon.

Please reopen the issue if 0.8.0 is not solving it.
